index-scheduler/src/lib.rs

```diff
@@ -794,6 +794,12 @@ impl IndexScheduler {
         Ok(res)
     }
 
+    // Return true if there is at least one task that is processing.
+    pub fn is_task_processing(&self) -> Result<bool> {
+        let processing_tasks = self.processing_tasks.read().unwrap().processing.len();
+        Ok(processing_tasks > 0)
+    }
```

Comment:
I like this new function, but you can use the `RoaringBitmap::is_empty` helper.
```suggestion
    pub fn is_task_processing(&self) -> Result<bool> {
        Ok(!self.processing_tasks.read().unwrap().processing.is_empty())
    }
```

Reply:
@Kerollmops 
Applied the suggestion in a new commit.